client/App.jsx

```diff
@@ -36,7 +36,7 @@ import Chat from './components/chat/Chat';
 import { WebSocketLink } from 'apollo-link-ws';
 
 const httpLink = new HttpLink({
-  uri: 'http://localhost:3000/graphql'
```

Comment:
This creates a breaking change in the addChapters feature. For temp fix we can revert this to using a fetch request on this feature.